.travis.yml

```diff
@@ -0,0 +1,51 @@
+sudo: required
+
+language: node_js
+
+node_js:
+  - "10.3.3"
+
+git:
+  depth: 2
+
+services:
+  - docker
+
+env:
+  global:
+    - IMAGE_REPO=gcr.io/dd-decaf-cfbf6/caffeine
+    - IMAGE_TAG=travis-ci-test
+    - IMAGE=${IMAGE_REPO}:${IMAGE_TAG}
+
+before_script:
+  - npm install -g @angular/cli
+
+install:
+  - npm install
+  - npm run build
+
+script:
```

Comment:
I'd call it `tests` or something like that.

Reply:
So this seems like a convention from `model` service. I'm okay with that in this case. One thing that would be amazing is to have the license validator script added to CI. No need to create a Makefile, just add the https://github.com/DD-DeCaF/model/blob/devel/scripts/verify_license_headers.sh file, and customize it to match ts, js, scss, css, sass, html. This could help you: https://unix.stackexchange.com/a/15309 but maybe there's an easier solution, I don't know.